test/integration/features/provision_various_versions.feature

```diff
@@ -0,0 +1,27 @@
+@provision-various-versions @openshift @minishift-only
+Feature: Provision all major Openshift versions
+  As a user I can provision major versions of openshift
+
+Scenario Outline:: Provision all major OpenShift versions
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --openshift-version <serverVersion>" succeeds
+     Then Minishift should have state "Running"
+     When executing "minishift openshift version" succeeds
+     Then stdout should contain
+     """
+     openshift <serverVersion>
+     """
+      And JSON config file "machines/minishift.json" contains key "OcPath" with value matching "<ocVersion>"
+     When executing "minishift oc-env" succeeds
+     Then stdout should contain
+      """
+      <ocVersion>
+      """
+     When executing "minishift delete --force" succeeds
```

Comment:
Little confusion...We are deleting the minishift instance not clearing the cache where oc is stored. Each time the loop runs it will keep the oc binary might causes conflict. To mke the clean run each time i would suggest ```When executing "minishift delete --force --clear-cache" succeeds``` would be a better resolution.

Reply:
@amitkrout I wanted to use the cached ISO to reduce execution time, so did not cleared the cache. Also if we do anot want to use cache in any test, we can clear cache first.

Reply:
> Each time the loop runs it will keep the oc binary that might causes conflict

@amitkrout it is not as you can see the test. Also it should not conflict.

Reply:
Got it...thanks for the clarification